Changing and Adding Properties/changing_properties.js

```diff
@@ -0,0 +1,24 @@
+const player1 = {
+	name: 'Ashley',
+	color: 'purple',
+	isTurn: true,
+	play: function () {
+		if (this.isTurn) {
+			console.log(`${this.name} is now playing!`);
+		}
+	}
+};
+
+const player2 = {
+	name: 'Guil',
+	color: 'red',
+	isTurn: false,
+	play: function () {
```

Comment:
We can practice **DRY** _(don't repeat yourself)_ here. Create a separate variable for the `play`.

Reply:
Thanks, @lycoangelo, This is updated now.